healthworkforce/features/utils.js

```diff
@@ -0,0 +1,283 @@
+'use strict'
+
+const axios = require('axios')
+const { expect } = require('chai')
+const nock = require('nock')
+
+const OPENHIM_PROTOCOL = process.env.OPENHIM_PROTOCOL || 'http'
+const OPENHIM_API_HOSTNAME = process.env.OPENHIM_API_HOSTNAME || 'localhost'
+const OPENHIM_TRANSACTION_API_PORT =
+  process.env.OPENHIM_TRANSACTION_API_PORT || '5001'
+
+const OPENHIM_API_PASSWORD =
+  process.env.OPENHIM_API_PASSWORD || 'instant101'
+const OPENHIM_API_USERNAME =
+  process.env.OPENHIM_API_USERNAME || 'root@openhim.org'
+
+const authHeader = new Buffer.from(
+  `${OPENHIM_API_USERNAME}:${OPENHIM_API_PASSWORD}`
+).toString('base64')
+
+const practitionerName = "Bob"
+const practitionerRoleId = "PR123566"
+const locationName = "GoodHealth Clinic"
+const organizationName = "Clinical Lab"
+
+exports.triggerSync = async () => {
+  await axios({
+    url: `${OPENHIM_PROTOCOL}://${OPENHIM_API_HOSTNAME}:${OPENHIM_TRANSACTION_API_PORT}/mcsd-trigger`,
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Basic ${authHeader}`
+    }
+  })
+  .then(() => {
+    console.log('Data successfully sent to dhis')
+  })
+  .catch((err) => {
+    console.log('Error in sending data to dhis', err.message)
+  })
+}
+
+exports.ihrisMockServicePractitioner = async () => {
+  await verifyResourceDoesNotExist('Practitioner')
+
+  nock('http://mock-service:4000')
```

Comment:
Like Martin was saying in the stand up, instead of using nock we should rather allow the mock service to reply to this request. Even if it just gives a simple static response.

Reply:
You might have to add the test resources to the data that the mock service is already returning so that the test still function. However, this is better because at least when the real services are available all the code stays the same and we just change the urls.